docs/manage/config-policies.rst

```diff
@@ -61,6 +61,14 @@ or CLI.
    It is recommended that administrators save their configuration policies for each scope in
    separate YAML or JSON files. This provides an easy way to back up and restore policies if needed.
 
+
+.. note::
+
+   Checkpoint storage can be set for workspaces and in experiment invariant config but the two 
+   settings have different uses. Checkpoint storage set for a workspace is a default that can be overridden 
+   by specifying checkpoint storage in experiment configuration. Experiment invariant config is 
+   non-overridable and will replace any workspace defaults as well as user-submitted experiment configs.
+
```

Comment:
release note description of workspace checkpoint storage:
Checkpoint storage can now be configured at a workspace level. Experiments created in projects will now inherit checkpoint storage configuration from the project's workspace if set. Experiment configuration can override the workspace level checkpoint storage configuration.

Reply:
Alternative suggestion:
Checkpoint storage can be configured for workspaces and in experiment invariant settings, but they have different functions. Workspace checkpoint storage serves as a default that can be overridden by experiment configurations, while experiment invariant settings are fixed and will override both workspace defaults and user-defined configurations.